Makefile

```diff
@@ -3,9 +3,57 @@ include ./Makefile.maven
 include ./Makefile.docker
 
 PROJECT_NAME ?= mqtt-bridge
+GITHUB_VERSION ?= main
```

Comment:
Is this used for anything?

Reply:
Good catch. I was getting code from the HTTP bridge. I should double check if it's used there somehow or we can remove it on both repos.

Reply:
@scholzj I noticed we are using this in the strimzi operator repo only to set a github version during the documentation generation with asciidoctor. Other repositories have the same env var declared but it's not used at all (I checked drain cleaner, http bridge, access operator). We should remove that env var from those repos. Removing from here for now.